### PR TITLE
Add content for "using gel electrodes" guide and Pro kit

### PR DESCRIPTION
**Pull request by bhawnasehgal**
@lorforlinux please review the changes [here](https://bhawnasehgal.github.io/upsidedownlabs.github.io/) and merge.

### CODE REVIEW

File: kits/index.rst [hunk 1]
```diff
@@ -1,8 +1,41 @@
...
+         :link: https://docs.upsidedownlabs.tech/kits/diy-neuroscience/basic/index.html
+
+      .. grid-item-card:: DIY Neuroscience Kit Pro
+         :text-align: center
+         :link: https://docs.upsidedownlabs.tech/kits/diy-neuroscience/pro/index.html
+
+.. only:: latex
+
+   1. `DIY Neuroscience Kit Basic <https://docs.upsidedownlabs.tech/kits/diy-neuroscience/basic/index.html>`_
+   2. `DIY Neuroscience Kit Pro <https://docs.upsidedownlabs.tech/kits/diy-neuroscience/pro/index.html>`_
```
Comment (lorforlinux) [inline kits/index.rst:37]: ref links please

**Review by lorforlinux — COMMENTED** [review]
@bhawnasehgal I have added some comments, please check and resolve.

File: guides/index.rst [hunk 2]
```diff
@@ -3,9 +3,36 @@
 Guides
 ######################
 
+In this section, you'll find easy-to-follow guides that will help you while using either BioAmp hardware or any of our DIY Neuroscience Kits.
```
[the remark this answers is not in the document]
Comment (bhawnasehgal) [reply guides/index.rst:6]: Done, resolving this conversation.

File: guides/index.rst [hunk 3]
```diff
@@ -3,9 +3,36 @@
...
+
+.. only:: html
+
+   .. grid:: 2 2 2 2
+      :margin: 4 4 0 0 
+      :gutter: 2
+
+      .. grid-item-card:: Skin preparation guide
+         :text-align: center
+         :link: https://docs.upsidedownlabs.tech/guides/usage-guides/skin-preparation/index.html
```
[the remark this answers is not in the document]
Comment (bhawnasehgal) [reply guides/index.rst:16]: Understood, making the changes

File: guides/index.rst [hunk 4]
```diff
@@ -3,9 +3,36 @@
...
+         :text-align: center
+         :link: https://docs.upsidedownlabs.tech/guides/usage-guides/using-bioamp-bands/index.html 
+
+      .. grid-item-card:: Guide to use gel electrodes
+         :text-align: center
+         :link: https://docs.upsidedownlabs.tech/guides/usage-guides/using-gel-electrodes/index.html 
+
+.. only:: latex
+
+   1. `Skin preparation guide <https://docs.upsidedownlabs.tech/guides/usage-guides/skin-preparation/index.html>`_
```
[the remark this answers is not in the document]
Comment (bhawnasehgal) [reply guides/index.rst:28]: We can skip creating PDFs for brief pages like these. Removing the latex code from guides, hardware, and DIY kits pages.

File: guides/usage-guides/using-gel-electrodes/index.rst [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+.. figure:: media/hydrogel-electrodes.*
+    :align: center
+
+2. Classification on the basis of connectivity
+================================================
+
+a. Connecting via snap
+-----------------------
+
+Some electrodes have a metallic snap connector on them which enables you to connect BioAmp Snap Cable by snapping the dry electrode part of the cable onto the electrode.
```
[the remark this answers is not in the document]
Comment (bhawnasehgal) [reply guides/usage-guides/using-gel-electrodes/index.rst:63]: Adding the BioAmp Snap cable link to our store for now. Created issue #47 for this

File: guides/usage-guides/using-gel-electrodes/index.rst [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+
+.. figure:: media/snap-connection.*
+    :align: center
+
+b. Connecting via tab
+-----------------------
+
+Some electrodes do not have the metallic snap connector on them, instead they have a conductive tab extending from the surface of the electrode. You can directly use
+BioAmp Alligator Cable to connect to the tab.
```
[the remark this answers is not in the document]
Comment (bhawnasehgal) [reply guides/usage-guides/using-gel-electrodes/index.rst:72]: Created issue #47 to resolve this

File: kits/index.rst (hunk 1 above)
Comment (bhawnasehgal) [reply kits/index.rst]: Okay

File: hardware/index.rst [hunk 7]
```diff
@@ -3,6 +3,61 @@
...
+
+.. only:: latex
+
+   1. `BioAmp EXG Pill <https://docs.upsidedownlabs.tech/hardware/bioamp/bioamp-exg-pill/index.html>`_
+   2. `Muscle BioAmp Shield <https://docs.upsidedownlabs.tech/hardware/bioamp/muscle-bioamp-shield/index.html>`_
+   3. `Muscle BioAmp Patchy <https://docs.upsidedownlabs.tech/hardware/bioamp/muscle-bioamp-patchy/index.html>`_
+   4. `Muscle BioAmp Blip <https://docs.upsidedownlabs.tech/hardware/bioamp/muscle-bioamp-blip/index.html>`_
+   5. `BioAmp v1.5 <https://docs.upsidedownlabs.tech/hardware/bioamp/bioamp-v1.5/index.html>`_
+   6. `Muscle BioAmp Candy <https://docs.upsidedownlabs.tech/hardware/bioamp/muscle-bioamp-candy/index.html>`_
+   7. `Muscle bioAmp BisCute <https://docs.upsidedownlabs.tech/hardware/bioamp/muscle-bioamp-biscute/index.html>`_
```
[the remark this answers is not in the document]
Comment (bhawnasehgal) [reply hardware/index.rst]: Made the changes

**Comment by bhawnasehgal** [comment]
Reopening the PR

File: guides/usage-guides/using-gel-electrodes/index.rst [hunk 8]
```diff
@@ -0,0 +1,112 @@
...
+.. warning:: For people having fragile skin, it is recommended to use either gel electrodes with hydrogel/standard adhesive or use `BioAmp Bands <https://docs.upsidedownlabs.tech/guides/usage-guides/using-bioamp-bands/index.html>`_.
+
+1. Skin Preparation
+======================
+
+Remove any excessive hair on the targeted area. Apply Nuprep skin preparation gel on the skin surface where electrodes would be placed to remove dead skin cells and clean the skin from dirt. After rubbing the skin surface thoroughly, clean it with an alcohol wipe or a wet wipe.
+
+For more information, please check out detailed step by step `skin preparation guide <https://docs.upsidedownlabs.tech/guides/usage-guides/skin-preparation/index.html>`_.
+
+.. note:: Always ensure that the prepared skin area is dry prior to applying the gel electrodes.
```
[the remark this answers is not in the document]
Comment (bhawnasehgal) [reply guides/usage-guides/using-gel-electrodes/index.rst:91]: I searched on the internet for the effect of alcohol swabs on the skin. Turns out it does leaves the skin dry, so adding the above note in the skin preparation guide

[the remark this answers is not in the document]
Comment (bhawnasehgal) [reply guides/usage-guides/using-gel-electrodes/index.rst:91]: We'll definitely test it out while performing experiments

**Pull request merged**